app/models/school.rb

```diff
@@ -32,4 +32,23 @@ def sierra_code
 
     return match.sierra_code
   end
+
+  
+  # Get all active schools
+  def self.active_schools
+    active.map do |school|
+      [school.name + school_code(school), school.id] 
+    end
+  end
+
+
+  def self.inactive_school(school)
+    ["[INACTIVE] #{school.name + school_code(school)}", school.id]
```

Comment:
This is more of a refactor than you were initially going for, but as we are repeating that `#{school.name + school_code(school)}` construction, maybe you could make a new instance method called, say, `label`, which would combine name and code, and could also add the `[INACTIVE]` label if the school isn't active.